make.ps1

```diff
@@ -4,7 +4,7 @@ Param(
     [String] $Target = "build",
     [String] $AdditionalArgs = '',
     [String] $Build = '',
-    [String] $VersionTag = '4.3-2',
+    [String] $VersionTag = '4.3-8',

```

Comment:
Rather than 4.3-7 ?

Reply:
That parameter is actually for THIS image, the last release for THIS image was 4.3-7, so the new release will be 4.3-8 for this image, it does not follow the base image version since there may be fixes here that don't require base version changes.